dataprep/eda/basic/compute.py

```diff
@@ -1032,6 +1058,127 @@ def calc_heatmap(
     return df_res, grp_cnt_stats
 
 
+def calc_stats_num(
+    srs: dd.Series, **kwargs: Any,
+) -> Tuple[Dict[str, str], Dict[str, str], Dict[str, str]]:
+    """
+    Calculate stats from a numerical column
+
+    Parameters
+    ----------
+    srs
+        a numerical column
+
+    Returns
+    -------
+    Tuple[Dict[str, Any], Dict[str, Any], Dict[str, Any]]
+        Dictionaries that contain Overview, Quantiles and Descriptives
+    """
+    size = len(srs)  # include nan
+    count = srs.count()  # exclude nan
+    uniq_count = srs.nunique()
+    inf_count = ((srs == np.inf) | (srs == -np.inf)).sum()
+    non_zero_count = np.count_nonzero(srs)
+    overview_dict = {
+        "Distinct Count": uniq_count,
+        "Unique (%)": uniq_count / count,
+        "Missing": size - count,
+        "Missing (%)": 1 - (count / size),
+        "Infinite": inf_count,
+        "Infinite (%)": inf_count / size,
+        "Mean": kwargs["mean"],
+        "Minimum": kwargs["min"],
+        "Maximum": kwargs["max"],
+        "Zeros": size - non_zero_count,
+        "Zeros (%)": 1 - non_zero_count / size,
+        "Memory Size": srs.memory_usage() / 1024,
+    }
+    quantiles_dict = {
+        "Minimum": kwargs["min"],
+        "5-th Percentile": kwargs["quantile"].iloc[5],
+        "Q1": kwargs["quantile"].iloc[25],
+        "Median": kwargs["quantile"].iloc[50],
+        "Q3": kwargs["quantile"].iloc[75],
+        "95-th Percentile": kwargs["quantile"].iloc[95],
+        "Maximum": kwargs["max"],
+        "Range": kwargs["max"] - kwargs["min"],
+        "IQR": kwargs["quantile"].iloc[75] - kwargs["quantile"].iloc[25],
+    }
+    descriptives_dict = {
+        "Standard Deviation": kwargs["std"],
+        "Coefficient of Variation": kwargs["std"] / kwargs["mean"],
+        "Kurtosis": kurtosis(srs, nan_policy="omit"),
+        "Mean": kwargs["mean"],
+        "Median Absolute Deviation": median_absolute_deviation(srs, nan_policy="omit"),
+        "Skewness": skew(srs, nan_policy="omit"),
```

Comment:
Skew didn't get formatted:
![Screen Shot 2020-05-23 at 4 28 50 PM](https://user-images.githubusercontent.com/25874021/82742421-b7645700-9d12-11ea-8a47-4cc645bccbda.png)
I think this can be fixed with `"Skewness": float(skew(srs, nan_policy="omit")),`